composer.json

```diff
@@ -5,6 +5,7 @@
     "keywords": ["toggl", "ajt", "api", "guzzle"],
     "homepage": "http://github.com/arendjantetteroo/guzzle-toggl",
     "license": "MIT",
+    "version": "9.0.0",
```

Comment:
i will probably release this as v2, not v9.

Reply:
Fair enough, I quite like locking module versions to their corresponding API version so it was a bit of an opinionated change